### DISCUSSION TITLE
i've been using notion-sdk-py, i'm intereted in finding more example codes for updating database, adding rows or columns to database

### DISCUSSION THREAD

**Discussion by tyehuoPHD** [jheddings/notional#127] [General]
i've been using notion-sdk-py, i'm intereted in finding more example codes for updating database, adding rows or columns to database

that's all.

**Comment by jheddings** [comment]
HI @tyePhDCandy !  Thanks for dropping by...

Have you browsed the examples folder?  There are several scripts that should help get you started.  For even more ideas, the unit tests cover most of the functionality in Notional.

Notion is not like a traditional database, so you never really add "rows" or "columns" to a database.  Instead, the database contains Pages and defines a property schema for all child pages.

Let me know if you run into any problems and I'll try to help you out.

**Reply by tyehuoPHD** [reply]
Thanks for the work. I will look into it. Good project!
